flexmeasures/data/services/scheduling.py

```diff
@@ -286,18 +299,19 @@ def make_schedule(
         rq_job.meta["data_source_info"] = data_source_info
         rq_job.save_meta()
 
-    ts_value_schedule = [
-        TimedBelief(
-            event_start=dt,
-            belief_time=belief_time,
-            event_value=-value,
-            sensor=sensor,
-            source=data_source,
-        )
-        for dt, value in consumption_schedule.items()
-    ]  # For consumption schedules, positive values denote consumption. For the db, consumption is negative
-    bdf = tb.BeliefsDataFrame(ts_value_schedule)
-    save_to_db(bdf)
+    for result in consumption_schedule:
+        ts_value_schedule = [
+            TimedBelief(
+                event_start=dt,
+                belief_time=belief_time,
+                event_value=value,
+                sensor=result["sensor"],
+                source=data_source,
+            )
+            for dt, value in result["data"].items()
+        ]  # For consumption schedules, positive values denote consumption. For the db, consumption is negative
```

Comment:
About the sign convention...

- [Scheduler internally] For values passed around within our scheduler internally: `consumption = +`
- [Scheduler inputs] For power values of inflexible devices searched in the db by our schedulers, we already look at the sensor's `consumption_is_positive` attribute ([see `data/models/planning/utils.py` lines 204-208](https://github.com/FlexMeasures/flexmeasures/blob/78e20233903440c1ce3e831a79152689c116f915/flexmeasures/data/models/planning/utils.py#L204-L208))
- [Scheduler outputs] For power values of flexible devices saved to the db by our schedulers, we should look at the sensor's `consumption_is_positive` attribute, too, rather than our previous assumption that we must flip. -> **todo**
- [UI] The UI always shows the same sign as the db
- [dev API] The dev API always returns data with the same sign as the db (e.g. the UI uses the dev API)
- [user API] The non-dev (e.g. v3) API always returns data with `consumption = +`. For this we should also be looking at the sensor's `consumption_is_positive` attribute, rather than our previous assumption that we must flip ([see `api/v3_0/sensors.py` line 523](https://github.com/FlexMeasures/flexmeasures/blob/78e20233903440c1ce3e831a79152689c116f915/flexmeasures/api/v3_0/sensors.py#L523)).-> **todo**

I think we can tackle both todos in this PR.